RELEASE/scripts/autoscend/paths/community_service.ash

```diff
@@ -1315,38 +1315,6 @@ boolean LA_cs_communityService()
 	familiar prior = my_familiar();
 	switch(curQuest)
 	{
-	case 0:
-		{
-			auto_log_info("Service Complete, finishing finish...", "blue");
-			try
-			{
-				if(do_cs_quest(30))
```

Comment:
Err, isn't this call right here still important?

Reply:
you are right. change made.